src/components/LessonInfoDisplay/LessonInfoDisplay.js

```diff
@@ -2,15 +2,21 @@ import React, { useState } from "react";
 import { Col, Row } from "reactstrap";
 import { compose } from "recompose";
 import { withRouter, Redirect } from "react-router-dom";
+import { withFirebase } from "utils/Firebase";
 import { useRef } from "react";
 import useOutsideClick from "../WordSelector/useOutsideClick";
 import "./LessonInfoDisplay.scss";
+import DeleteConfirmation from "../../components/DeleteConfirmation/DeleteConfirmation";
 
 function LessonInfoDisplay(props) {
+  const { firebase } = props;
```

Comment:
should destructure props i.e. `const {firebase, handleDeletedLesson, setDisplay, ...} = props`